Mesh_3/include/CGAL/Mesh_3/Refine_facets_3.h

```diff
@@ -356,6 +360,19 @@ class Refine_facets_3_base
     return sstr.str();
   }
 
+  int dimension(const Intersection& intersection) const
+  {
+    return std::get<2>(intersection);
+  }
+  Index index(const Intersection& intersection) const
+  {
+    return std::get<1>(intersection);
+  }
+  Point_3 point(const Intersection& intersection) const
+  {
+    return std::get<0>(intersection);
+  }
```

Comment:
Here the point is returned by copy. Maybe we could return the ref:

```suggestion
  const Point_3& point(const Intersection& intersection) const
  {
    return std::get<0>(intersection);
  }
```